challenge.js

```diff
@@ -34,12 +34,17 @@
  *	Using the same HTML elements as above...
  *
  *	Suppose A is the body element, B, C, and D are all divs,
- *		and E is a p. 
+ *		and E is a p.
  *
  *	Write a CSS selector that can be used to retrieve E from the DOM.
  *
  *	Answer below.
- */
+ */<body>
+      <div>
+          <p>
+          </p>
+      </div>
+  </body>
```

Comment:
You modeled it (correctly) as HTML. The question was asking for a CSS selector string. 1/4